### PR TITLE
Improve Boss

### PR DESCRIPTION
**Pull request by GabrielRavier**
This makes the style consistent in a few files (comment and conditionals spacing mostly)

### CODE REVIEW

File: src/BossAlmo1.cpp [hunk 1]
```diff
@@ -13,7 +13,8 @@
 
 static void ActBossChar_Core_Face(NPCHAR *npc)
 {
-	RECT rect[4] = {
+	RECT rect[4] =
+	{
```
Comment (Clownacy) [inline src/BossAlmo1.cpp]: That's my coding style. I do something similar with typedef'd structs.

Comment (GabrielRavier) [reply src/BossAlmo1.cpp]: Oh OK, so I should go revert all these types (declaration brace style) of changes across all the PRs?

Comment (Clownacy) [reply src/BossAlmo1.cpp]: Yeah

**Comment by GabrielRavier** [comment]
I think I addressed the RECT problem from your review

File: src/BossAlmo1.cpp [hunk 2]
```diff
@@ -114,7 +114,8 @@ static void ActBossChar_Core_Mini(NPCHAR *npc)
 	int ym;
 	int xm;
 
-	RECT rect[3] = {
+	RECT rect[3] =
+	{
```
Comment (Clownacy) [inline src/BossAlmo1.cpp]: Missed one

Comment (GabrielRavier) [reply src/BossAlmo1.cpp]: Solved it I think

**Comment by GabrielRavier** [comment]
I think I solved them all now

**Pull request merged**